src/ProgressBar/ProgressBar.js

```diff
@@ -793,6 +793,8 @@ module.exports = kind(
 		// of the label determination could help here - rjd
 		{path: ['progress', 'popup', '$.popupLabel.content'], method: 'ariaValue'},
 		{path: ['accessibilityValueText'], method: function () {
+			this.setAriaAttribute('aria-valuetext', null);
+			this.setAriaAttribute('aria-valuenow', null);
```

Comment:
Actually, it's probably better to collapse the `accessibilityValueText` observer into `ariaValue()`. We could then drop the observer in moonstone/Slider and just override `ariaValue()` there.